addon/components/rdf-input-fields/currency-input.hbs

```diff
@@ -0,0 +1,40 @@
+{{#unless @inTable}}
+  <AuLabel
+    @error={{this.hasErrors}}
+    @warning={{this.hasWarnings}}
+    @required={{this.isRequired}}
+    for={{this.inputId}}
+  >
+    {{@field.label}}
+  </AuLabel>
+{{/unless}}
+
+<div class="CurrencyInput">
+  <AuInput
+    @error={{this.hasErrors}}
+    @disabled={{@show}}
+    @width={{if @inTable "block"}}
+    @warning={{this.hasWarnings}}
+    id={{this.inputId}}
+    value={{this.value}}
+    {{on "blur" this.updateValue}}
+    {{au-inputmask
+      options=(hash
+        alias="currency"
+        radixPoint=","
+        groupSeparator="."
+        allowMinus=true
+        inputType="decimal"
```

Comment:
I took a quick look at the docs, and this doesn't seem to be mentioned as an option? https://robinherbots.github.io/Inputmask/#/documentation/numeric#inputtype

Maybe I'm missing something, the docs are a bit vague.

Reply:
I was very confused yesterday with the `inputmode` or `inputType` in the examples but now it's clear. You are right. I removed the `inputType` as the default is already `text` and `inputmode` is `decimal` as default so that can also stay.